### PR TITLE
Add Import setting for Spoke products

### PR DESCRIPTION
**Pull request by apreed**
This addresses Issue #250 

*Description of changes:*
* Renamed ImportIntoSpokeLocalPortfolioTask to CopyIntoSpokeLocalPortfolioTask
* Created new task ImportIntoSpokeLocalPortfolioTask which uses AssociateProductWithPortfolio API rather than CopyProduct
* Create new setting in spoke-local-portfolios to choose copy or import - defaults to current behaviour Copy if setting does not exist
* Updated docs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/puppet/sharing_a_portfolio.rst [hunk 1]
```diff
@@ -56,6 +57,33 @@ The valid values for regions are:
 - list of AWS regions - you can type in a list of AWS regions (each region selected should be present in your config)
 
 
+What is the setting for Products Copy or Import?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+    This was added in version 0.72.0, previous versions always Copy
```
Comment (apreed) [inline docs/source/puppet/sharing_a_portfolio.rst]: This version number will need to be changed subject to release

**Comment by apreed** [comment]
I haven't gone as far as creating a new manifest block, the setting remains in spoke-local-portfolios.
The reason for this is that the new functionality is still dealing with spoke local portfolios - it's just adding a new method to get products into those portfolios. Creating a new block called "spoke-import-portfolios" is misleading, as it implies this is importing the whole portfolio, which is not true.

**Comment by eamonnfaherty** [comment]
Thanks for this. 

Is it ready for testing?

**Comment by apreed** [comment]
It is, make test passes but I haven't updated the coverage, let me know if I can help out with that. If your local tests work out then if you can build me an issue specific deployable then I will do some QA.

**Pull request merged**